contracts/contracts/GrantRoundManager.sol

```diff
@@ -141,12 +145,13 @@ contract GrantRoundManager is SwapRouter {
       require(_donations[i].grantId < registry.grantCount(), "GrantRoundManager: Grant does not exist in registry");
 
       // Used later to validate ratios are correctly provided
-      donationRatios[_donations[i].token] += _donations[i].ratio;
+      donationRatios[_donations[i].token] = donationRatios[_donations[i].token].add(_donations[i].ratio);
```

Comment:
why are we doing this change ?

Reply:
We used to get SafeMath for free with 0.8.x, but 0.7.6 needs explicit usage of SafeMath, which is what we're doing here

Reply:
Oh derp ! Gotcha